pulsar-broker/src/main/java/org/apache/pulsar/broker/cache/LocalZooKeeperCacheService.java

```diff
@@ -144,10 +144,11 @@ private void initZK() throws PulsarServerException {
      */
     @SuppressWarnings("deprecation")
     public CompletableFuture<Optional<LocalPolicies>> createPolicies(String path, boolean readFromGlobal) {
-        checkNotNull(path, "path can't be null");
-        checkArgument(path.startsWith(LOCAL_POLICIES_ROOT), "Invalid path of local policies");
-
         CompletableFuture<Optional<LocalPolicies>> future = new CompletableFuture<>();
+        if (path == null || !path.startsWith(LOCAL_POLICIES_ROOT)) {
```

Comment:
actually, this method returns `CompletableFuture<>` so, we should fail the future rather throwing runtime exception.